controllers/tweet-controller.js

```diff
@@ -146,6 +146,24 @@ const tweetController = {
     } catch (err) {
       next(err)
     }
+  },
+  replyTweet: async (req, res, next) => {
+    try {
+      const UserId = Number(helpers.getUser(req).id)
+      const TweetId = Number(req.params.tweet_id)
+      if (!TweetId) return res.status(400).json({ status: 'error', message: 'TweetId is required' })
+
+      const tweet = await Tweet.findOne({ where: { id: TweetId } })
+      if (!tweet) return res.status(404).json({ status: 'error', message: 'Tweet is not found' })
+
+      const { comment } = req.body
+      if (!comment || !comment.trim()) return res.status(400).json({ status: 'error', message: 'Reply is blank.' })
```

Comment:
I think it's better to use `reply` instead of `comment`, or at least use same name in req.body and in res.message. It's might be confusing to discuss an attribute in two different name.

Reply:
Oh~ my mistake. The comment attribute is named by test file.